lib/buildAccount.ts

```diff
@@ -93,105 +85,145 @@ async function saveAccountToConfig({
   return validName;
 }
 
-async function buildNewAccount({
-  name,
-  accountType,
-  accountConfig,
-  env,
-  portalLimit, // Used only for developer test accounts
-  force = false,
-}) {
+export async function buildDeveloperTestAccount(
```

Comment:
I decided to split `buildNewAccount` into two separate functions. It adds a few extra lines of code, but IMO is much easier to read. I don't think there was a ton of utility in combining them previously, since it only combined two account types and would have been super messy if we ever attempted to add a third. It also was more or less impossible to safely type